package.json

```diff
@@ -9,6 +9,7 @@
     "ionicons": "^3.0.0",
     "react": "16.0.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-24.0.0.tar.gz",
+    "react-native-collapsible": "^0.10.0",
```

Comment:
Collapsible library. Totally open to using a different one if we find a better one. It works, but it's not as smooth as I'd like for the animation. Good enough for now... but we may want to use another or build our own eventually.